bluebrain/repo-patches/packages/arm-forge/package.py

```diff
@@ -6,13 +6,15 @@ class ArmForge(BuiltinArmForge):
     __doc__ = BuiltinArmForge.__doc__
 
     version(
-        "21.1.2-Linux-x86_64",
-        sha256="ebc99fa3461d2cd968e4d304c11b70cc8d9c5a2acd68681cec2067c128255cd5",
-        url="https://content.allinea.com/downloads/arm-forge-21.1.2-linux-x86_64.tar",
+        "22.0.1-Linux-x86_64",
+        sha256="8f8a61c159665d3de3bc5334ed97bdb4966bfbdb91b65d32d162d489eb2219ac",
+        url="https://content.allinea.com/downloads/arm-forge-22.0.1-linux-x86_64.tar",
     )
 
-    version(
-        "20.2.0-Redhat-7.0-x86_64",
-        sha256="26592a77c42f970f724f15b70cc5ce6af1078fd0ef9243a37c3215916cfa7cf4",
-        url="https://content.allinea.com/downloads/arm-forge-20.2-Redhat-7.0-x86_64.tar",
-    )
+    def setup_run_environment(self, env):
+        # ENV variables required to avoid licensing and SLURM issues on BB5
+        env.unset('HTTP_PROXY')
+        env.unset('HTTPS_PROXY')
+        env.unset('http_proxy')
+        env.unset('https_proxy')
+        env.set('ALLINEA_USE_SSH_STARTUP', "1")
```

Comment:
The issue for the `module` error was due to using an integer `1` directly, instead of a `"1"` string.  This is something that the logs of the CI already showed:

```
==> Warning: Could not write module file [/gpfs/.../modules/linux-rhel7-skylake/arm-forge/22.0.1-Linux-x86_64]
==> Warning:    --> expected string or bytes-like object <--
```

With the help of @pramodk, we have confirmed that the module is working locally, so this PR is ready for review and deployment.